File: platforms/ios/foundation/Sources/NativeblocksFoundation/FoundationProvider.swift
import Foundation
import NativeblocksRuntime

public class FoundationProvider {

    public static func provide(name: String = "default") {
        NativeblocksFoundationBlockProvider.provideBlocks(name: name)
        NativeblocksFoundationModifierProvider.provideModifiers(name: name)
        NativeblocksFoundationTypeProvider.provideTypes(name: name)
    }
}

File: platforms/android/foundation/foundation/src/main/java/io/nativeblocks/foundation/FoundationProvider.kt
package io.nativeblocks.foundation

import io.nativeblocks.foundation.integration.consumer.block.FoundationBlockProvider
import io.nativeblocks.foundation.integration.consumer.modifier.FoundationModifierProvider

object FoundationProvider {
    fun provide(instanceName: String = "default") {
        FoundationTypeProvider.provideTypes(instanceName = instanceName)
        FoundationBlockProvider.provideBlocks(instanceName = instanceName)
        FoundationModifierProvider.provideModifiers(instanceName = instanceName)
    }
}